frontend/src/screens/Settings.tsx

```diff
@@ -0,0 +1,128 @@
+import React, { useContext } from 'react'
+import { CommonActions } from '@react-navigation/native'
+import { View, StyleSheet } from 'react-native'
+import {
+  Appbar,
+  Dialog,
+  Paragraph,
+  Portal,
+  Button,
+  Card,
+  Switch,
+} from 'react-native-paper'
+import { authContext } from '../providers/auth.provider'
+
+const Settings = ({ navigation, route }): JSX.Element => {
+  const { logout } = useContext(authContext)
+  const [visible, setVisible] = React.useState(false)
+  const [isSwitchOn, setIsSwitchOn] = React.useState(true)
+
+  const onToggleSwitch = () => setIsSwitchOn(!isSwitchOn)
+
+  const showDialog = () => setVisible(true)
+
+  const hideDialog = () => setVisible(false)
+
+  const handleLogOut = (): void => {
+    logout().then(() => {
```

Comment:
Added a logout function. Misschien moet die nog veranderd worden. hij staat in de authstate